joomla/joomla-cms
# 4.0 dev

### Summary of Changes
For A11Y reasons, the Cancel button in the registration form should be removed. There is no reason to have it, anyway, as users can just use the browser 'Back' button or navigate elsewhere, without submitting the form. In any case, using a link, disguised as a button is not the best practice and causes A11Y issues.


### Testing Instructions
Open the registration form. There should be only a submit button and not a cancel button.


### Expected result
Better A11Y support.


### Actual result
Better A11Y support.
